frontend/components/Spinner/_styles.scss

```diff
@@ -61,6 +45,38 @@
   }
 }
 
+.loading-spinner.small {
+  background: none;
+  box-shadow: none;
+  height: auto;
+  margin: 0;
+  transform: scale(0.7);
```

Comment:
```
  .loading-spinner__ring {
    width: 26px;
    height: 26px;

    div {
      width: 26px;
      height: 26px;
    }
  }
```

Height of spinner should be changed to 18px instead of current 24px

Reply:
I tried matching the pixel dimensions from Figma, but the spinner looked much too small in the buttons. I think because in the app the buttons are 38px tall, but in Figma they are 34px. 

I'm going to propose to @mike-j-thomas we make them slightly bigger, as currently implemented. I can fine-tune with him in the code since the Figma pixel dimensions aren't matching 1 for 1.